### PR TITLE
New version: Stellar.StellarCLI version 22.0.0-rc.1

### PR DESCRIPTION
**Pull request by fnando**
Checklist for Pull Requests
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/191259)

### CODE REVIEW

**Comment by fnando** [comment]
@microsoft-github-policy-service agree

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Executable C:\Program Files (x86)\Stellar CLI\stellar.exe returned exit code: -1073741515

Automated error analysis suggests -1073741515 may mean STATUS_DLL_NOT_FOUND - possibly due to a dependency on `Microsoft.VCRedist.2015+.x64` (or `.x86`).

(Automated response - build 897.)

**Comment by stephengillie** [comment]
HI @fnando,

Manual Validation ended with:
image

Is there a way to bypass this prompt and complete the install silently?

File: manifests/s/Stellar/StellarCLI/22.0.0-rc.1/Stellar.StellarCLI.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.6.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Stellar.StellarCLI
+PackageVersion: 22.0.0-rc.1
+InstallerType: nullsoft
+Installers:
```
Comment (stephengillie) [inline manifests/s/Stellar/StellarCLI/22.0.0-rc.1/Stellar.StellarCLI.installer.yaml:7]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
Manual Validation ended with:
image

**Comment by fnando** [comment]
@stephengillie thank you for the review. I ended up replacing NSIS with Inno Setup, which worked without any dependencies like Microsoft.VCRedist.2015+.x64 (I've tested without having any of this installed).

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Executable C:\Program Files (x86)\Stellar CLI\stellar.exe returned exit code: -1073741515

Automated error analysis suggests -1073741515 may mean STATUS_DLL_NOT_FOUND - possibly due to a dependency on `Microsoft.VCRedist.2015+.x64` (or `.x86`).

(Automated response - build 897.)

**Comment by stephengillie** [comment]
Manual Validation ended with:
> 
> 2024-11-14 15:14:23.438 [FAIL] Installer failed security check. Url: https://github.com/stellar/stellar-cli/releases/download/v22.0.0-rc.1/stellar-cli-installer-22.0.0-rc.1-x86_64-pc-windows-msvc.exe Result: 0x80004005

(Automated response - build 897.)

File: manifests/s/Stellar/StellarCLI/22.0.0-rc.1/Stellar.StellarCLI.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+PackageLocale: en-US
+Publisher: Stellar
+PublisherUrl: https://stellar.org/
+PackageName: Stellar CLI
+PackageUrl: https://github.com/stellar/stellar-cli
+License: Apache 2.0
+LicenseUrl: https://github.com/stellar/stellar-cli/blob/main/LICENSE
+ShortDescription: CLI for Stellar developers
+Description: CLI for Stellar developers
+Moniker: soroban
```
Comment (leighmcculloch) [inline manifests/s/Stellar/StellarCLI/22.0.0-rc.1/Stellar.StellarCLI.locale.en-US.yaml:15]: Unclear why we need to mention soroban here. Soroban is the execution runtime that the Stellar network uses. It's not built into the Stellar-CLI, and the Stellar-CLI is primarily about interacting with the Stellar network, not the execution runtime. A while ago the CLI carried the name "soroban-cli" due to historical hang over.

File: manifests/s/Stellar/StellarCLI/22.0.0-rc.1/Stellar.StellarCLI.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# Created with komac v2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Stellar.StellarCLI
+PackageVersion: 22.0.0-rc.1
+PackageLocale: en-US
+Publisher: Stellar
```
Comment (leighmcculloch) [inline manifests/s/Stellar/StellarCLI/22.0.0-rc.1/Stellar.StellarCLI.locale.en-US.yaml:7]: @fnando I think the publisher would be the full foundation's name.
```suggestion
Publisher: Stellar Development Foundation
```

**Review by leighmcculloch — COMMENTED** [review]
Two comments.

**Comment by stephengillie** [comment]
Verify:
image

**Review by stephengillie — APPROVED** [review]

**Comment by fnando** [comment]
@stephengillie this should be available by now, right?

```
 winget source update && winget search --id Stellar.StellarCLI
Updating all sources...
Updating source: msstore...
Done
Updating source: winget...
  █████████████████████████████▋  99%
Done
No package found matching input criteria.
```

**Comment by stephengillie** [comment]
Hi @fnando,

Sorry for not replying earlier. It probably needed to wait for the next Publish pipeline run to complete.

**Pull request merged**